extensions/amp-analytics/0.1/vendors.js

```diff
@@ -1594,6 +1594,40 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
     },
   },
 
+  'shinystat': {
```

Comment:
Would you mind moving this up so that the configs are alphabetical? I know the two or three above are not sorted but we'll fix those separately.

Reply:
I moved it between "segment" and "snowplow". (rebased)